src/components/menubar/menubar.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import acmhack from './acmhack.svg';
+import hackLogo from './hack-logo.png';
```

Comment:
SVG please – see `hack-logo.svg` in https://github.com/uclaacm/Styleguide/blob/master/assets/downloads/hack-logomarks.zip.